src/main/webapp/app/exercises/shared/exercise-hint/shared/code-hint-cast.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { CodeHint } from 'app/entities/hestia/code-hint-model';
+import { ExerciseHint } from 'app/entities/hestia/exercise-hint.model';
+
+/**
+ * Pipe to transform an ExerciseHint into a CodeHint
+ */
+@Pipe({
+    name: 'cast',
```

Comment:
I think we need to choose a different name here. ``cast`` is too generic for this very specific operation. Maybe just ``castToCodeHint`` or something like this.